Sources/StreamChatUI/UIConfig+MessageComposer.swift

```diff
@@ -41,7 +41,7 @@ public extension _UIConfig {
 
         /// Buttons used in the left side of the composer. Corresponds to attachment and actions buttons.
         public var composerButton: UIButton.Type = UIButton.self
-        public var textView: _ChatMessageComposerInputTextView<ExtraData>.Type = _ChatMessageComposerInputTextView<ExtraData>.self
+        public var textView: _ChatInputTextView<ExtraData>.Type = _ChatInputTextView<ExtraData>.self
```

Comment:
Same here, this should go to the root of UIConfig, and probably rename to `inputTextView` ?